docs/docsite/rst/playbooks_best_practices.rst

```diff
@@ -293,21 +293,21 @@ What about just reconfiguring my webservers?::
 
     ansible-playbook -i production webservers.yml
 
-What about just my webservers in Boston?::
+What about just my webservers in Boston?:
 
     ansible-playbook -i production webservers.yml --limit boston
 
-What about just the first 10, and then the next 10?::
+What about just the first 10, and then the next 10?
    
     ansible-playbook -i production webservers.yml --limit boston[1:10]
     ansible-playbook -i production webservers.yml --limit boston[11:20]
 
-And of course just basic ad-hoc stuff is also possible.::
+And of course just basic ad-hoc stuff is also possible:
```

Comment:
s/:/::/

Reply:
The reason for the double colons is because those are code blocks below it.  However, rst is going to leave a single colon in the html output which will probably look funky since these are questions (and thus you end up with "?:" in the output).  For the ones that end in questions we probably should spell out that it's a code-block.  For the ones that naturally end in a colon we can continue to use double colon (although we might want to use the long form of specifying a code-block if it syntax highlights the output.)  So these should be like this (my indentation is going to be hell because this is a proportional font but hopefully you can get the idea:
```
What about just reconfiguring my webservers?

 .. code-block:: shell
 
     ansible-playbook -i production webservers.yml
 
What about just my webservers in Boston? 
 .. code-block:: shell

    ansible-playbook -i production webservers.yml --limit boston
 
What about just the first 10, and then the next 10?

.. code-block: shell

     ansible-playbook -i production webservers.yml --limit boston[1:10]
     ansible-playbook -i production webservers.yml --limit boston[11:20]
 
And of course just basic ad-hoc stuff is also possible::

     ansible boston -i production -m ping